Cargo.toml

```diff
@@ -129,3 +129,7 @@ inherits = "release"
 codegen-units = 1
 lto = true
 strip = "symbols"
+
+[[bin]]
+name = "1-0-0-beta-31"
+path = "src/migrations/1.0.0-beta.31.rs"
```

Comment:
We should add migrations as a subcommand because it makes it much more convenient to use in our released Docker image.

Reply:
I don't think that's worth the updated complexity. The inflation of the binary is probably negligible.

Reply:
We most definitely will have migrations in the future so a `migrate` subcommand makes sense in any case IMO. The only thing that this subcommand is then supposed to do is to check whether a certain migration is applicable by checking the versions and then execute the migration binary, or deny it with an appropriate message.

Reply:
It's maybe negligible for the first migration :trollface:

Reply:
The other option is to create/find an implementation of blake2b that we can use in a mongo script.

Reply:
@Alex6323 found a JavaScript one at some point. But I think we should just use our in-memory implementation.